libopae/plugins/xfpga/sysfs.c

```diff
@@ -1137,11 +1190,15 @@ fpga_result get_port_sysfs(fpga_handle handle, char *sysfs_port)
 
```

Comment:
Gets the port sysfs path, given a handle to the corresponding FME?  Maybe add a comment that describes this.

Reply:
👍